### PR TITLE
Update Contracts with ContenType fetching from facade

### PR DESCRIPTION
**Pull request by CDimonaco**

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 linters-settings:
   gomoddirectives:
     replace-allow-list:
-      - github.com/trento-project/contracts
+      - github.com/trento-project/contracts/go
```
Comment (arbulu89) [inline .golangci.yaml:4]: I think we can remove the `linters-settings` block as we don't have any replace anymore

File: internal/factsengine/mapper.go [hunk 2]
```diff
@@ -130,7 +130,11 @@ func FactsGatheredToEvent(gatheredFacts entities.FactsGathered) ([]byte, error)
 		FactsGathered: facts,
 	}
 
-	eventBytes, err := events.ToEvent(&event, entities.FactsGathererdEventSource, uuid.New().String())
+	eventBytes, err := events.ToEvent(
+		&event,
+		events.WithSource(entities.FactsGathererdEventSource),
+		events.WithID(uuid.New().String()),
```
Comment (arbulu89) [inline internal/factsengine/mapper.go:136]: If we don't use the `WithID`, the uuid will be generated in the contract side in the same way.

**Review by arbulu89 — APPROVED** [review]
Great!
I would remove the `linters-settings` block in the golang ci.
Green light besides that

File: internal/factsengine/mapper.go (hunk 2 above)
Comment (CDimonaco) [reply internal/factsengine/mapper.go:136]: Yes it's true, I wanted just to point it out as a matter of clarity, anyway we can discuss further on the sensible details of the contracts :D

File: .golangci.yaml (hunk 1 above)
Comment (dottorblaster) [reply .golangci.yaml:4]: I think @arbulu89 is right ._.

Comment (CDimonaco) [reply .golangci.yaml]: done!

**Pull request merged**